source/ItemInfoDisplay.cpp

```diff
@@ -98,20 +91,8 @@ void ItemInfoDisplay::DrawAttributes(const Point &topLeft) const
 
 void ItemInfoDisplay::DrawTooltips() const
 {
-	if(!hoverCount || hoverCount-- < HOVER_TIME || !hoverText.Height())
-		return;
-
-	Point textSize(hoverText.WrapWidth(), hoverText.Height() - hoverText.ParagraphBreak());
-	Point boxSize = textSize + Point(20., 20.);
-
-	Point topLeft = hoverPoint;
-	if(topLeft.X() + boxSize.X() > Screen::Right())
-		topLeft.X() -= boxSize.X();
-	if(topLeft.Y() + boxSize.Y() > Screen::Bottom())
-		topLeft.Y() -= boxSize.Y();
-
-	FillShader::Fill(topLeft + .5 * boxSize, boxSize, *GameData::Colors().Get("tooltip background"));
-	hoverText.Draw(topLeft + Point(10., 10.), *GameData::Colors().Get("medium"));
+	tooltip.Draw();
+	tooltip.DecrementCount();
```

Comment:
Can we decrement this only if `hasHover` is false, and avoid the duplicated increment this way?

Reply:
`hasHover` is only false if you're hovering over one of the buttons in the shop.

Reply:
I did try to remove all the places that were double incrementing the hover count, but this was the one place where it isn't really easy to do. At least it can't be done by changing ItemInfoDisplay alone. It might be that OutfitInfoDisplay and ShipInfoDisplay need to increment/decrement the counter.

Reply:
Upon further review, it still seems like more trouble than it's worth. It's how it was working before anyway.